lib/matplotlib/text.py

```diff
@@ -1466,7 +1470,9 @@ def _get_xy_transform(self, renderer, s):
         elif isinstance(s, Transform):
             return s
         elif not isinstance(s, str):
-            raise RuntimeError(f"Unknown coordinate type: {s!r}")
+            raise RuntimeError(
```

Comment:
Ditto.

Reply:
Nope. You cannot get `(str, str)` and `callable` out of `check_is_instance()`.

Reply:
Good point.